bin/deploy

```diff
@@ -204,16 +205,16 @@ main() {
       fi
       ARGS+=('--name' "$LSST_CONDA_ENV_NAME")
       for c in $LSST_CONDA_CHANNELS; do
-        ARGS+=("-c $c")
+        ARGS+=("-c$c")
```

Comment:
Does `—channel conda-forge` work? Would look better at least.

Reply:
That should be two dashes. Stupid autocorrect.

Reply:
It does not.